### PR TITLE
Added writeup post

### PR DESCRIPTION
**Pull request by sidntrivedi**

### CODE REVIEW

**Comment by sidntrivedi** [comment]
@ryzokuken @vaibhavk Please merge this so that we can complete the writeup.

File: package.json [hunk 1]
```diff
@@ -32,6 +32,7 @@
   "devDependencies": {
     "eslint": "^4.19.1",
     "eslint-plugin-react": "^7.7.0",
+    "gatsby-new-blog": "^1.0.4",
```
Comment (ryzokuken) [inline package.json]: Why do we need this?

**Review by ryzokuken — APPROVED** [review]
Looks great, just need a minor clarification.

File: package.json (hunk 1 above)
Comment (sidntrivedi) [reply package.json]: Oops..was adding a package to automate the process of writing a new blog post. Removing it for now.

**Comment by sidntrivedi** [comment]
@ryzokuken Done. PTAL.

**Review by ryzokuken — APPROVED** [review]
Thanks

**Pull request merged**